### PR TITLE
Is telephone validation

### PR DESCRIPTION
**Pull request by claranceliberi**

### CODE REVIEW

**Review by makuzaverite — COMMENTED** [review]
Thanks for this PR

FYI we shall merge this once the new error handling mechanism lands into the project

**Review by PatrickNiyogitare28 — APPROVED** [review]

**Review by makuzaverite — APPROVED** [review]
I'm merging this for now thanks for the great work

**Pull request merged**
